libraries/APM_OBC/APM_OBC.cpp

```diff
@@ -225,10 +256,12 @@ APM_OBC::check(APM_OBC::control_mode mode, uint32_t last_heartbeat_ms, bool geof
     case STATE_GPS_LOSS:
         if (!gcs_link_ok) {
             // losing GCS link when GPS lock lost
-            // leads to termination
-            if (!_terminate) {
-                GCS_MAVLINK::send_statustext_all(MAV_SEVERITY_CRITICAL, "Dual loss TERMINATE");
-                _terminate.set(1);
+            // leads to termination if AFS_DUAL_LOSS is 1
+            if(_enable_dual_loss) {
```

Comment:
None other than personal preference - I can make them inline for the next commit.